src/main/java/org/alfresco/heartbeat/SystemUsageDataCollector.java

```diff
@@ -84,6 +111,40 @@ public void afterPropertiesSet() throws Exception
 
         Runtime runtime = Runtime.getRuntime();
         Map<String, Object> systemUsageValues = new HashMap<>();
+
+        // operating system MBean info
+        Map<String, Object> cpu = new HashMap<>();
+        OperatingSystemMXBean osMBean = ManagementFactory.getPlatformMXBean(OperatingSystemMXBean.class);
+        if (osMBean != null)
+        {
+            if (osMBean instanceof UnixOperatingSystemMXBean)
+            {
+                long openFileDescriptorCount = ((UnixOperatingSystemMXBean) osMBean).getOpenFileDescriptorCount();
+                systemUsageValues.put("openFileDescriptorCount", new Long(openFileDescriptorCount));
+            }
+
+            // processor info
+            double processCpuLoad = osMBean.getProcessCpuLoad() * 100;
```

Comment:
Let's use an integer here.  A double can be very accurate and we'll end up sending long json values with little point.